src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
wildcard case would just not filter, or only blacklist known-issue elements, I guess.

Maybe it's not worth worrying about it and just land it, @lgalfaso what do you think?